l10n_it_account_stamp/__manifest__.py

```diff
@@ -3,11 +3,12 @@
 # Copyright 2018 Ermanno Gnan (ermannognan@gmail.com)
 # Copyright 2018 Lorenzo Battistini (https://github.com/eLBati)
 # Copyright 2018 Sergio Zanchetta (https://github.com/primes2h)
+# Copyright 2019 Stefano Consolaro (https://github.com/mymage)
 # License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
 
 {
-    'name': 'Italian Localization - Tax Stamp',
-    'version': '11.0.1.0.0',
+    'name': 'Italian Localization - Imposta di bollo',
+    'version': '11.0.1.1.1',
     'category': 'Localization/Italy',
     'summary': 'Tax stamp automatic management',
```

Comment:
Ciao, come da linee guida https://www.odoo-italia.org/documentazione/sviluppo/guidelines.html

s/Italian Localization/ITA
e il `summary` andrebbe in italiano.